01-gabbar/00-tekton-pipelines/00-build/values.yaml

```diff
@@ -65,6 +65,13 @@ pipeline-charts:
         runAfter:
         - build-and-push
         - stakater-sonarqube-scanner-v1
+      - taskName: stakater-cosign-signing-v1
+        runAfter:
```

Comment:
this tasks doesnt exist in default tasks in pipeline chart, you can see the tasks that are currently available [here.](https://github.com/stakater/pipeline-charts/tree/main/pipeline-charts/default-config/tasks). 
you ll need to specify its params,workspaces, when conditions in this file under taskName.